website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -20,6 +20,31 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## October 2024
 
+<Expandable alt_header="Coalesce 2024 announcements">
+
+  Documentation for new features and functionality announced at Coalesce 2024:
+
+  - Iceberg table support for [Snowflake](https://docs.getdbt.com/reference/resource-configs/snowflake-configs#iceberg-table-format)
+  - [Athena](https://docs.getdbt.com/reference/resource-configs/athena-configs) and [Teradata](https://docs.getdbt.com/reference/resource-configs/teradata-configs) adapter support in dbt Cloud.
+  - dbt Cloud now hosted on [Azure](https://docs.getdbt.com/docs/cloud/about-cloud/access-regions-ip-addresses)
```

Comment:
do we want to open these links in new tabs? flagging bc links in previous release notes just use for example `/docs/cloud/<page>`

Reply:
Yeah, Since we're not staggering these, there are a lot of items to scan to prevent a lot of back-and-forth and page reloads if they're interested in multiple.